pytential/symbolic/elasticity.py

```diff
@@ -0,0 +1,166 @@
+__copyright__ = "Copyright (C) 2021 Isuru Fernando"
```

Comment:
I think it'd be good to have `CalculusPatch` tests to ensure that PDE rediual for Stokes and elasticity is zero.

Reply:
Added a test to check the stokes PDE in a9931e50073a09766de1c724f3d9ad6037b11f17 and elasticity in bb314713c1ac24cae66d582bcc8fd775b5bb4ad4